src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -5033,7 +5033,10 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
     <value>Invalid name for a preprocessing symbol; '{0}' is not a valid identifier</value>
   </data>
   <data name="ERR_FeatureNotAvailableInVersion7_1" xml:space="preserve">
-    <value>Feature '{0}' is not available in C# 7.1.  Please use language version {1} or greater.</value>
+    <value>Feature '{0}' is not available in C# 7.1. Please use language version {1} or greater.</value>
+  </data>
+  <data name="ERR_LanguageVersionCannotHaveLeadingZeroes" xml:space="preserve">
+    <value>Specified language version '{0}' cannot have leading zeroes</value>
```

Comment:
Per last email from @gafter, we are accepting this as a breaking change, and putting it in the Post VS2017 breaking change markdown (edited in this PR).

---
In reply to: [107026072](https://github.com/dotnet/roslyn/pull/17990#discussion_r107026072) [](ancestors = 107026072)